### PR TITLE
Fix x-kubernetes-list-map-keys

### PR DESCRIPTION
**Pull request by apelisse**
/assign @lavalamp

### CODE REVIEW

**Comment by apelisse** [comment]
Related to https://github.com/kubernetes/kubernetes/pull/84113/

/hold

We need to agree on what values are accepted for structType and mapType: "separate" or "map"

**Comment by apelisse** [comment]
I think we agree on the value for structType/mapType to granular!

/hold cancel

File: keps/sig-api-machinery/0006-apply.md [hunk 1]
```diff
@@ -172,11 +172,10 @@ It is defined either in Go IDL by pefixing the list with `//
...
 - Associative lists: Kubernetes has a pattern of using lists as dictionary, with
 "name" being a very common key. People can now reproduce this pattern by using
-`// +listType=associative`, or in the OpenAPI with
-`"x-kubenetes-list-type": "map"` along with `//
-+x-kubernetes-list-map-key=name`, or `"listMapKey": "name"`. Items
-of an associative lists are owned by the person who applied the item to the
-list.
+`// +listType=associative`, or in the OpenAPI with `"x-kubenetes-list-type":
+"map"` along with `"x-kubernetes-list-map-keys": ["name"]`, or `"listMapKey":
+"name"`. Items of an associative lists are owned by the person who applied the
```
Comment (lavalamp) [inline keps/sig-api-machinery/0006-apply.md]: Should the second one be `// +listMapKey=name`?

Comment (apelisse) [reply keps/sig-api-machinery/0006-apply.md]: :-/ Fixed.

**Pull request merged**